Convex_hull_3/doc/Convex_hull_3/CGAL/Convex_hull_traits_3.h

```diff
@@ -8,11 +8,12 @@ The class `Convex_hull_traits_3` serves as a traits class for the function
 function when `R` is a kernel with exact predicates but inexact constructions 
 (note that the type `Plane_3` is a triple of `Point_3` and not `R::Plane_3`). 
 
+\tparam MFG must be a ,model of  MutableFaceGraph
 \cgalModels `ConvexHullTraits_3`
 \cgalModels `IsStronglyConvexTraits_3` 
 
 */
-template< typename R >
+  template< typename R, typename MFG = Polyhedron_3<R> >
```

Comment:
Only looking at the diff on github, it seems that the function convex_hull_3(begin, end, P) has not been updated so that the polyhedron type is passed to the traits. `Default_traits_for_Chull_3` should have an extra template parameter to be able to pass the `FaceGraph` model to `Convex_hull_traits_3`.